packages/api/src/command/medical/document/document-reconvert.ts

```diff
@@ -0,0 +1,318 @@
+import { DocumentReference } from "@medplum/fhirtypes";
+import { resourceTypeForConsolidation } from "@metriport/api-sdk";
+import { S3Utils } from "@metriport/core/external/aws/s3";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { out } from "@metriport/core/util/log";
+import { capture } from "@metriport/core/util/notifications";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { groupBy } from "lodash";
+import { DocRefMapping } from "../../../domain/medical/docref-mapping";
+import { convertCDAToFHIR } from "../../../external/fhir-converter/converter";
+import { getDocuments as getDocumentsFromFHIRServer } from "../../../external/fhir/document/get-documents";
+import { downloadedFromHIEs } from "../../../external/fhir/shared";
+import { getMetriportContent } from "../../../external/fhir/shared/extensions/metriport";
+import { PatientModel as Patient } from "../../../models/medical/patient";
+import { Config } from "../../../shared/config";
+import { errorToString } from "../../../shared/log";
+import { formatNumber } from "../../../shared/numbers";
+import { getDocRefMappings } from "../docref-mapping/get-docref-mapping";
+import { deleteConsolidated as deleteConsolidatedOnFHIRServer } from "../patient/consolidated-delete";
+import { getConsolidated } from "../patient/consolidated-get";
+import { getPatientOrFail } from "../patient/get-patient";
+import { makeDocRefContentToFileFunction, SimplerFile } from "./document-query-storage-info";
+
+dayjs.extend(duration);
+
+// TODO review this
```

Comment:
nit: mention specific issue # here?

Reply:
no follow-up issue needed, just wanted to review this to make sure we wanted to run one at a time, removing the comment